usagov/vote-gov-drupal
# Vote 1094/state link translations

Jira ticket (required)
[VOTE-1094](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-1094&assignee=609161faf6c0960069bf61b3)

Description (optional)
This adds translated strings of state page links and moves the link placement to be more descriptive for accessibility.

Testing steps
New state pages should have links that read as follows:
Online registration is currently not available. (Find out ways to register) to vote at %state_name%’s state election website.
Register in person at your local election office. (Learn more about how to register) on %state_name%’s state election website.
Voter registration is not required in Virginia. (Learn more about voting) on %state_name%’s state election website
(Start your online registration) on %state_name%’s state election website
You can also (register to vote by mail or in person) on %state_name%’s state election website
You can (confirm your voter registration status) on %state_name%’s state election website
Languages other than English should include (in English) translated at the end of the link.
